### ISSUE TITLE
Bug jail for 15.8

### ISSUE THREAD

**Issue by PatoBeltran** [NuGet/Home#6889]
* [x] - [Feedback][Accessibility] Error messages are unreadable in Package manager console because of font color and background color - [Internal 489860](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/489860)
* [x] - [Feedback] [Test Failure] Incorrect “latest stable version” show in “Version” drop-down of “Installed”/“Consolidate” tab in a solution have multiple projects - [Internal 491404](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/491404)
* [x] - [Test Failure][Accessibility] The selected text in PMC with the color theme: “Blue (Extra Contrast)” is invisible - [Internal 481498](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/481498)
* [x] - NuGetVSExtension.NuGetPackage loads ExtensionsExplorerUI.dll before VS starts - [Internal 497251](https://devdiv.visualstudio.com/DevDiv/NuGet/_workitems/edit/497251)

* [x] - [Feedback][Accessibility] NuGet CTRL+E Keyboard shortcut doesn't work - [Internal 469569](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/469569)
* [x] -Status message for "Clear All NuGet Cache(s)" can be hidden on window resize -  https://github.com/NuGet/Home/issues/5938
* [x] - [Test Failure][Accessibility] String ‘Prerelease’ under package button is covered by its package description in PM UI - https://github.com/NuGet/Home/issues/4504
* [x] - [Test Failure][Accessibility] Package source drop down and settings button truncated when selecting ‘Microsoft Visual Studio Offline Packages’ in PM UI - https://github.com/NuGet/Home/issues/4502
* [x] - Package manager UI : preview changes, ok button not automatically useable by keyboard - https://github.com/NuGet/Home/issues/6893
* [x] - Nuget Package Manager Settings: the 1st added source isn’t editable until the focus is moved out -  [Internal 620255](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/620255)
* [x] - Nuget Package Manager Settings: double click of OK button is required to save changes - [Internal 620257](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/620257)
* [x] - NuGet Package Manager: UI Artifacts during package installation - [Internal 620265](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/620265)
* [x] - Unable to install NuGet package to .NET Core App after uninstalling - [Internal 615109](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/615109)
* [x] - Usability: VS internal windows don't have horizontal scroll when their width is not enough to display all controls - [Internal 620261](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/620261)
* [x] - "Do not show this again" checkbox has no context - [Internal 503976](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/503976)
* [x] - NuGet 3.0 JAWS: cannot narrate the title, texts inside, and buttons of License Acceptance dialog -  [Internal 208064](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/208064)
 * [x] - NuGet 3.0 JAWS: cannot narrates the install/uninstall preview dialog - [Internal 208060](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/208060)
* [x] - [Accessibility]: NuGet Uninstall: Preview Changes dialog should tell the user something about the large list box in Narrator - [Internal 519030](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/519030)
* [x] - NuGet 3.0 JAWS: cannot narrates the controls/texts from the File Conflict dialog - [Internal 208062](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/208062)

**Moved to appropriate area:**
* [x] - [Feedback][Accessibility] nuget text box cut and paste clipboard - [Internal 546415](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/546415)
* [x] - Search dropdown does not show recent queries in NuGet Packages Manager window - [Internal 620319](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/620319)

**Moved to appropriate milestone:**
* [x] - NuGet Package Manager: confusing buttons state after a package install/uninstall - [Internal 620278](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/620278)
